Assets/Scripts/Game/MagicAndEffects/EntityEffectManager.cs

```diff
@@ -869,8 +870,13 @@ public void UseItem(DaggerfallUnityItem item, ItemCollection collection = null)
                             collection.RemoveItem(item);
                         }
                     }
-
-                    break;
+                }
+                else if (enchantment.type == EnchantmentTypes.SpecialArtifactEffect) // For artifact items
+                {
+                    if (!GameManager.Instance.EntityEffectBroker.GetArtifactBundleSettings(out bundleSettings, enchantment.param))
+                        continue;
+                    bundle = new EntityEffectBundle(bundleSettings, entityBehaviour);
+                    AssignBundle(bundle, AssignBundleFlags.ShowNonPlayerFailures);
                 }
```

Comment:
I modified this same area in https://github.com/Interkarma/daggerfall-unity/pull/1233 so there is likely to be a merge conflict.